test/functional/p2p_compactblocks.py

```diff
@@ -135,11 +131,18 @@ def make_utxos(self):
         block2.vtx.append(tx)
         block2.hashMerkleRoot = block2.calc_merkle_root()
         block2.solve()
-        self.test_node.send_and_ping(msg_block(block2))
+        self.segwit_node.send_and_ping(msg_block(block2))
         assert_equal(int(self.nodes[0].getbestblockhash(), 16), block2.sha256)
         self.utxos.extend([[tx.sha256, i, out_value] for i in range(10)])
         return
 
+    # Send balance to a segwit output
+    def make_segwit_output(self, node):
```

Comment:
Nit: Since `self` is not used -- this could be a function instead of a method?

Reply:
Eliminated this in the latest version.